pkg/integrations/manager_test.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/prometheus/prometheus/pkg/relabel"
 	"github.com/stretchr/testify/require"
 	"go.uber.org/atomic"
-	"gopkg.in/yaml.v3"
+	"gopkg.in/yaml.v2"
```

Comment:
question: was this a concious change? Why did we change it?

Reply:
Yes :) All of the other code imports `gopkg.in/yaml.v2` and I don't want to have dependencies on both. There's some slight behavioral differences that makes v2 preferable (like strict unmarshaling). Some imports of v3 still slip in, and I'm changing them as I catch them.

Reply:
OK, thanks. I have something similar happening in the redis PR. I'll try and address it.